inferentia/scripts/setup.sh

```diff
@@ -151,25 +135,25 @@ make triton-python-backend-stub -j16
 # since we need to use pip to update: 
 #  https://aws.amazon.com/blogs/developer/neuron-conda-packages-eol/
 pip config set global.extra-index-url https://pip.repos.neuron.amazonaws.com
-conda config --env --add channels https://conda.repos.neuron.amazonaws.com
 
+python -m pip install -U pip
+# Set pip repository pointing to the Neuron repository 
+python -m pip config set global.extra-index-url https://pip.repos.neuron.amazonaws.com
 if [ $USE_TENSORFLOW -eq 1 ]; then
-    conda install tensorflow-neuron pillow -y
-    # Update Neuron TensorBoard
-    pip install --upgrade tensorboard-plugin-neuron
     # Update Neuron TensorFlow
     if [ $TENSORFLOW_VERSION -eq 1 ]; then
-        pip install --upgrade tensorflow-neuron==1.15.5.* neuron-cc "protobuf<4"
+        # Install TensorFlow Neuron
+        python -m pip install tensorflow-neuron[cc]==1.15.5.* "protobuf"
     else
-        pip install --upgrade tensorflow-neuron[cc] "protobuf<4"
+        # Install TensorFlow Neuron
+        python -m pip install tensorflow-neuron[cc] "protobuf"
     fi
 fi
 
 if [ $USE_PYTORCH -eq 1 ]
 then
-    conda install torch-neuron torchvision -y
-    # Upgrade torch-neuron and install transformers
-    pip install --upgrade torch-neuron neuron-cc[tensorflow] "protobuf<4" torchvision "transformers==4.6.0"
+    # Install PyTorch Neuron
+    python -m pip install torch-neuron neuron-cc[tensorflow] "protobuf" torchvision
```

Comment:
Should we pin the version of these libraries and maybe update them when we have tested with new packages?

Reply:
I will defer this to @jbkyang-nvi as she removed the version pin in the reference PR.

However, I don't see it to be a major issue, because the protobuf and other libraries are for Inferentia use (relying on Inferentia validates against latest dependencies). And I don't think it will interfere with our use of protobuf (we build statically and thus no symbol expose to/depend on external protobuf), unless the Python side boilerplate of the backend uses Python protobuf?

Reply:
I believe this is the recommended way of installing the dependencies according to the [neuron documentation](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/quick-start/torch-neuron.html). The backend doesn't use protobuf. I would prefer to not pin libraries unless we want to stick to a specific version of inferentia.